File: dinner/Dinner/gcd.py
minimum = 0
gcd = 1
first_number = 12
second_number = 8

if first_number < second_number:
	minimum = first_number


else:minimum = second_number

for count in range(1, minimum + 1):
	if first_number % count == 0 and first_number % count == 0:
		gcd = count

print(gcd)

File: dinner/Dinner/GCD.js
let minimum = 0;
let GCD = 1;
let firstNumber = 12;
let secondNumber = 8;

if(firstNumber < secondNumber){
minimum = firstNumber;
}

else{minimum = secondNumber;}

for(int counter = 1; counter <= minimum; counter++){
if(firstNumber % counter == 0 && secondNumber % counter == 0){GCD = counter;}


}

console.log(GCD)